includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -733,8 +736,10 @@ function( $scope ) {
 		 * @since 1.6.0 The $token_response parameter was added.
 		 *
 		 * @param array $token_response Token response data.
+		 * @param string[] $scopes List of scopes.
+		 * @param string[] $previous_scopes List of previous scopes.
```

Comment:
We should also add a `@since` annotation documenting what version the scope params were added.

Reply:
👍